### ISSUE TITLE
Input image size inconsistency of the Mask RCNN model

### ISSUE THREAD

**Issue by 594zyc** [alfworld/alfworld#6]
Hello, 

Thanks for the open-sourced code and models! 

I have a question regarding the pretrained mask RCNN model. As in "augment_trajectories.py", I see you set the image size to 400x400 for generating the object detection dataset to fine-tune the mask RCNN. While in the MaskRCNNAgent, the input of the model is the original ALFRED frames which are of size 300x300.  I am wondering is there any risk of a detection performance loss with this inconsistency of input image sizes?

Thanks,
Yichi

**Comment by MohitShridhar** [comment]
Good question! During pre-training, we saved 400x400 images because objects like pens and pencils have really tiny masks that barely contain a few pixels. It's possible that the downsized 300x300 input could lead to missed objects. But it's tradeoff between how fast you want to run the simulator (AI2THOR) vs. the resolution of the image observations. 

On a side note, in this [video](https://www.youtube.com/watch?v=eHdn7bryKyI&feature=emb_title), the detector trained on 400x400 images was directly used with 1280x720 images. But the simulator took several hours to render these sequences.

You can also try out different input sizes by changing `['env']['thor']['screen_width']` and `['env']['thor']['screen_height']` in the [eval_config.yaml](https://github.com/alfworld/alfworld/blob/master/agents/config/eval_config.yaml) file for evaluation experiments.

**Comment by 594zyc** [comment]
Thanks so much for the quick response! This information is very helpful.
